scheduler/kubernetes/encrypted-volumes.md

```diff
@@ -0,0 +1,176 @@
+---
+layout: page
+title: "Encrypted volumes using PVC"
+keywords: portworx, container, kubernetes, storage, k8s, flexvol, pv, persistent disk, encryption, pvc
+sidebar: home_sidebar
+meta-description: "Looking to use a encrypted volume with Kubernetes? Follow this step-by-step tutorial on how to provision encrypted volumes with Kubernetes."
+---
+
+* TOC
+{:toc}
+
+This document describes how to provision an encrypted volume using Kubernetes and Portworx. For more information on encryption refer the [Encrypted Volumes page](/manage/encrypted-volumes.html).
+
+Before you start using PVC encryption, you need to setup a secrets provider to store your secret keys and configure Portworx to use it. Refer the Encryption section for more details on setting up various secret stores.
+
+### Using Storage Class
+You can set parameter in portworx Storage Class which tells Portworx that all PVCs created using this Storage Class need to be encrypted. Portworx uses the cluster wide secret to encrypt all the volumes creating using the secure storage classes. Refer [setting cluster wide secret](/secrets/portworx-with-kubernetes-secrets.html#setting-cluster-wide-secret-key), if you are using Kubernetes Secrets. You can check other secret store integrations in the Encryption section.
```

Comment:
Instead of redirecting (`Refer [setting cluster wide secret](/secrets/portworx-with-kubernetes-secrets.html#setting-cluster-wide-secret-key)`) embed the content right here. Will be easier to users to go over everything on one page.

You can use the includes approach to have common content and have multiple pages reference that.

Reply:
I think let's keep links here because those steps are specific to K8s secrets. Many people are still using vault for encryption. Let's keep links so that they can go to their respective provider. Added explicit link for k8s secrets to help new customers get started.